rust/etw_helpers/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "etw_helpers"
+version = "0.1.0"
+edition = "2021"
+resolver = "2"
+license = "MIT"
+rust-version = "1.67"
+authors = ["Kyle Sabo", "Microsoft"]
+repository = "https://github.com/microsoft/etw"
+
+[features]
```

Comment:
Should be a "flume" feature here too... I wonder where it went.

Reply:
Futures, Crossbeam, and Flume all implement channels, but some people have strong opinions on not wanting to include any code that uses on or another.